src/Raven.Studio/wwwroot/App/views/manage/adminLogsSettingsDialog.html

```diff
@@ -0,0 +1,56 @@
+<div class="modal-dialog modal-lg" role="document" id="adminLogsSettings">
+    <div class="modal-content" tabindex="-1">
+        <div class="modal-header">
+            <button type="button" class="close" data-bind="click: cancel" aria-hidden="true">
+                <i class="icon-cancel"></i>
+            </button>
+            <h4 class="modal-title">Admin Log Files Settings on Disk</h4>
+        </div>
+        <div class="modal-body">
+            <form class="form-horizontal" role="form">
+                <div class="form-group">
+                    <label class="col-md-3 control-label">Log Files Path:</label>
+                    <div class="col-md-9">
+                        <p class="form-control-static" data-bind="text: fullPath"></p>
+                    </div>
+                </div>
+                <div class="form-group">
+                    <label class="col-md-3 control-label">Retention Time:</label>
+                    <div class="col-md-9">
+                        <p class="form-control-static" data-bind="text: retentionTimeText"></p>
+                    </div>
+                </div>
+                <div class="form-group">
+                    <label class="col-md-3 control-label">Retention Size:</label>
+                    <div class="col-md-9">
+                        <p class="form-control-static" data-bind="text: retentionSizeText"></p>
+                    </div>
+                </div>
+                <div class="form-group">
+                    <label class="col-md-3 control-label">Log Mode:</label>
+                    <div class="col-md-9">
+                        <div class="dropdown">
+                            <button type="button" class="btn btn-default dropdown-toggle" data-toggle="dropdown" title="Set logs mode" id="selectedLogMode">
```

Comment:
why do we need margin-top: 5px here?

Reply:
well - it wasn't aligned enough to my taste